style.css

```diff
@@ -194,4 +177,21 @@ h2 a:visited {
 
 .article-tag {
     border: 0;
+}
+
+/* form  */
+
+.filtering-form {
+    border-radius: 0.3em;
+    background-color: var(--gray-6);
+    border: 1px solid var(--gray-3);
+    margin-left: 1.5em;
+}
+
+.form-input-size {
+    width: 180px;
+}
+
+.search-input-size {
```

Comment:
Remove `size` from this class name, it's not ok to have it here. What if we were to change 2 properties, size and background. Would you call it `search-input-size-background`?

Calling it `search-input` should be fine, i think.